projects/plugins/jetpack/_inc/lib/class.color.php

```diff
@@ -93,16 +93,19 @@ public function fromRgbInt( $red, $green, $blue ) {
 		if ( $red < 0 || $red > 255 ) {
 			throw new RangeException( 'Red value ' . $red . ' out of valid color code range' );
 		}
+		$red = intval( $red );
```

Comment:
So... I know it's only the bottom `$this->color =` line that's causing the warning, but how about we just group these together and put them all at the top of the function for improved readability?

Reply:
I don't know about the top - I wanted them to work after we have checked the colors for sane values. What about just doing `intval` directly in the color assignment line?

Reply:
Yeah you're right, we should either bunch them up but after all the conditions or yeah, you can just add the `intvals()` in the color assignment line to keep things clean :).